go-controller/pkg/ovn/policy.go

```diff
@@ -473,7 +486,10 @@ func (oc *Controller) createDefaultDenyMulticastPolicy() error {
 
 	// Remove old multicastDefaultDeny port group now that all ports
 	// have been added to the clusterPortGroup by WatchPods()
-	deletePortGroup("mcastPortGroupDeny")
+	err = deletePortGroup(oc.ovnNBClient, "mcastPortGroupDeny")
```

Comment:
would be nice if this was a constant, if you don't mind cleaning that up while you are here

Reply:
This is weird.  I don't see where the string "mcastPortGroupDeny" is used anywhere else in the code, so I don't know how that port group would have gotten created in the first place.  I do see it in a fake command in a pods_test.go, but the function that contains it never gets called.  I guess it should just be deleted?

Reply:
ah this is legacy cleanup from commit 40a90f06644a0d829343d6bed1a5f4a6156facdf

Could add a constant to types/const.go that's like LegacyMulticastPortGroupDeny or something. Otherwise I'm fine with it as is.

Reply:
I'm putting the constant with the others at the top of the policy.go if that's okay.